app/javascript/components/admin/UserModal.js

```diff
@@ -54,23 +80,26 @@ class UserModal extends React.Component {
         </Modal.Header>
         <Modal.Body>
           <Form>
-            <Form.Group>
-              <Form.Label className="input-label">Email Address</Form.Label>
+            <Form.Group controlId="email-input">
+              <Form.Label className="input-label">Email Address{schema?.fields?.email?._exclusive?.required && '*'}</Form.Label>
```

Comment:
```suggestion
              <Form.Label className="input-label">Email Address{schema?.fields?.email?._exclusive?.required && ' *'}</Form.Label>
```

Reply:
I went back and forth with this one and personally prefer it without the space (see screenshots).  @hackrm thoughts from the PO side?

<img width="512" alt="Screen Shot 2022-01-26 at 8 15 01 AM" src="https://user-images.githubusercontent.com/35042815/151169549-1038972b-81b7-4b2b-b768-1d8ddf9557a5.png">

<img width="499" alt="Screen Shot 2022-01-26 at 8 15 27 AM" src="https://user-images.githubusercontent.com/35042815/151169551-afc3a58e-15b0-4fd6-8ad1-57df742ef406.png">

Reply:
this was more for consistency sake because most of these in the app includes a space

Reply:
@sgober I could go either way on whether to include the space or not stylisticly speaking. But for this, I would opt for consistency with the rest of the application and add the space in since that is how we handle `*` throughout most of the application.

Reply:
Okay, so after DMing with @sgober a bit - we found that all modals in Sara Alert do not have the space before the `*`. But in the Enrollment wizard, that is where the space is present before the `*`. So since this is a modal, this should be consistent with the styling of the other modals, so **no space should be added here**.